fvm/evm/evm_test.go

```diff
@@ -223,32 +433,148 @@ func TestCadenceOwnedAccountDeploy(t *testing.T) {
 					let cadenceOwnedAccount <- EVM.createCadenceOwnedAccount()
 					cadenceOwnedAccount.deposit(from: <-vault)
 
-					let address = cadenceOwnedAccount.deploy(
-						code: [],
-						gasLimit: 53000,
-						value: EVM.Balance(attoflow: 1230000000000000000)
+					let bal = EVM.Balance(0)
```

Comment:
```suggestion
					let bal = EVM.Balance(attoflow: 0)
```

Reply:
This is a breaking change in Cadence 1.0. The named attribute is always required.